admissioncontroller/k8s/deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
             drop:
               - ALL
               - NET_RAW
-        image: eurogig/whorf@sha256:5e8fbbda58cce84226c4ddcdff50a28e3806252b5c5af1de9d82565e7be44dc0
+        image: bridgecrew/whorf
```

Comment:
@schosterbarak Do we want to add a tag here? I'd vote yes

Reply:
how would i bump it?

Reply:
sed...

Reply:
can you assis?

Reply:
Did we solve this?  I do a bump to the latest in another repo like this.  Might be hackish.  
```
        name: Build and push
        id: docker_build
        uses: docker/build-push-action@v2
        with:
          file: clust3rf8ck/Dockerfile
          push: true
          tags: eurogig/clust3rf8ck:${{ steps.get-latest-tag.outputs.tag }}, eurogig/clust3rf8ck:latest
      -
        name: Image digest
        run: echo ${{ steps.docker_build.outputs.digest }}
      - name: Update deployment
        uses: jacobtomlinson/gha-find-replace@master
        with:
          find: "- image: eurogig/clust3rf8ck@sha256:[a-f0-9]{64}"
          replace: "- image: eurogig/clust3rf8ck@${{ steps.docker_build.outputs.digest }}"
          include: "clust3rf8ck/clust3rf8ck.yaml"`

Reply:
Looks really nice, we did not solve this yet @eurogig